src/m.jl

```diff
@@ -22,7 +22,7 @@ end
 """
 mxvg
 
-@deprecate mxvg(m1, v2) m1 * v2
+@deprecate mxvg(m1, v2) m1 .* v2
```

Comment:
This change is incorrect. `.*` is element-wise multiplication but this should be a dot product, i.e. `*`.